src/styles/_catalog.scss

```
@@ -14,9 +14,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
-  }
+    transition-duration: 0.3s;
```

it will be correct to set not only the time, but also the transition-properties  to which a transition effect should be applied.
```suggestion
    transition-duration: color 0.3s, transform 0.3s;
```